From ea9a8672a19588e89c0bda8e0c31424687c1e05f Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Thu, 3 Nov 2016 12:14:56 -0700
Subject: [PATCH] Fix build with ICC on macOS: <atomic> is properly supported

I'm guess I introduced the regression in the commit
18ed6f20ad4327fa2ffbb687ac2f3ca7429c4348, which I wasn't sure about.

Change-Id: Ic46ff326a6ba46bc877cfffd14839f84fdf796e7
Reviewed-by: Jake Petroules <jake.petroules@qt.io>
--- src/corelib/global/qcompilerdetection.h.orig	2017-02-10 22:14:51 UTC
+++ src/corelib/global/qcompilerdetection.h
@@ -1066,12 +1066,6 @@
 #  ifndef _HAS_VARIADIC_TEMPLATES
 #   undef Q_COMPILER_VARIADIC_TEMPLATES
 #  endif
-# elif defined(_LIBCPP_VERSION)
-// libc++ uses __has_feature(cxx_atomic), so disable the feature if the compiler
-// doesn't support it. That's required for the Intel compiler 14.x or earlier on OS X, for example.
-#  if !__has_feature(cxx_atomic)
-#   undef Q_COMPILER_ATOMICS
-#  endif
 # endif
 # if defined(Q_COMPILER_THREADSAFE_STATICS) && defined(Q_OS_MAC)
 // Apple's low-level implementation of the C++ support library
